editoast/src/views/timetable.rs

```diff
@@ -351,6 +359,29 @@ async fn conflicts(
     Ok(Json(conflict_detection_response.conflicts))
 }
 
+pub fn filter_core_work_schedule(
+    ws: &WorkSchedule,
+    start_time: DateTime<Utc>,
+) -> crate::core::stdcm::WorkSchedule {
+    crate::core::stdcm::WorkSchedule {
+        start_time: elapsed_since_time_ms(&ws.start_date_time, &start_time),
+        end_time: elapsed_since_time_ms(&ws.end_date_time, &start_time),
+        track_ranges: ws
+            .track_ranges
+            .iter()
+            .map(|track| UndirectedTrackRange {
+                track_section: track.track.to_string(),
+                begin: (track.begin * 1000.0) as u64,
+                end: (track.end * 1000.0) as u64,
+            })
+            .collect(),
+    }
+}
+
+fn elapsed_since_time_ms(time: &NaiveDateTime, zero: &DateTime<Utc>) -> u64 {
```

Comment:
`zero` means 0 in my mind. Maybe we can find another name? How about `since` (because it's in the name of the function)?

Reply:
I changed it by `start_time`

Reply:
I changed it by `start_time`